saleor/graphql/schema.graphql

```diff
@@ -4582,6 +4584,7 @@ type ShippingMethod implements Node & ObjectWithMetadata {
   price: Money
   maximumOrderPrice: Money
   minimumOrderPrice: Money
+  zipCodes: [ShippingMethodZipCode]
```

Comment:
IMO these are not just zip codes, but zip code rules. For now, we have exclusion rules, in the future maybe also inclusion rules. I would consider renaming this field to `zipCodeRules: [ZipCodeRule]`, and accordingly, the mutations and types. But maybe this is fine, what you think @patrys @korycins ?